.github/workflows/static.yml

```diff
@@ -0,0 +1,76 @@
+# Simple workflow for generating and deploying static html in readability folder to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'apple/**'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'apple/**'
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  generate-distiller-output:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Generate distiller output for readability
+        run: |
+          go install github.com/omnivore-app/go-domdistiller@latest
+          for f in packages/readabilityjs/test/test-pages/*; do
+            echo "Processing $f file..."
+            # skip newsletters for now
+            if [ "$f" = "packages/readabilityjs/test/test-pages/newsletters" ]; then
+              continue
+            fi
+            go-domdistiller file -i $f/source.html -o $f/distiller.html
+          done
+  generate-static:
+    needs: generate-distiller-output
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Generate static html
+        run: |
+          cd packages/readabilityjs/test
+          python generate_static.py
+  deploy:
```

Comment:
Cool trick, I didn't know you can do this in an action

Reply:
I'm not sure if it will work because the workflow failed somewhere before